Backpack-SwiftUI/Calendar/Classes/CalendarDayCells/CalendarSelectableCell.swift

```diff
@@ -18,111 +18,11 @@
 
 import SwiftUI
 
-struct CalendarSelectableCell: View {
-    let selectionType: CalendarSelectionType
-    let calendar: Calendar
-    let accessibilityProvider: CalendarAccessibilityConfiguration
-    let dayDate: Date
-    let onSelection: (Date) -> Void
+struct CalendarSelectableCell<Cell: View>: View {
```

Comment:
nothing to see here, just reverting back to the old implementation, which supports better composition of views among other things